docs/Deployment.md

```diff
@@ -105,10 +105,10 @@ Before creating the cluster, the first node must be launched. Initially, a node
 
 We can validate our node is indeed `uninitialized` as follows:
 
-    $ redis-cli --raw -p 5001 RAFT.INFO
+    $ redis-cli -p 5001 INFO redisraft
```

Comment:
I wonder if we should register under module name `raft` to make this shorter. This would also be more consistent with the remaining `RAFT.` commands.

Reply:
Changed it to "raft". Also, looks like other modules are also using without 'redis'. So, we are in line with them.